app/components/polaris/autocomplete_component.rb

```diff
@@ -51,14 +54,18 @@ def popover_arguments
         full_width: true,
         inline: false,
         wrapper_arguments: {
-          data: {polaris_autocomplete_target: "popover"}
+          data: { polaris_autocomplete_target: "popover" }
```

Comment:
Fixes standardrb code style complains:
```suggestion
          data: {polaris_autocomplete_target: "popover"}
```